pkg/api/password.go

```diff
@@ -13,22 +14,22 @@ import (
 
 func SendResetPasswordEmail(c *models.ReqContext, form dtos.SendResetPasswordEmailForm) response.Response {
 	if setting.LDAPEnabled || setting.AuthProxyEnabled {
-		return response.Error(401, "Not allowed to reset password when LDAP or Auth Proxy is enabled", nil)
+		return response.Error(http.StatusUnauthorized, "Not allowed to reset password when LDAP or Auth Proxy is enabled", nil)
 	}
 	if setting.DisableLoginForm {
-		return response.Error(401, "Not allowed to reset password when login form is disabled", nil)
+		return response.Error(http.StatusUnauthorized, "Not allowed to reset password when login form is disabled", nil)
 	}
 
 	userQuery := models.GetUserByLoginQuery{LoginOrEmail: form.UserOrEmail}
 
 	if err := bus.Dispatch(&userQuery); err != nil {
 		c.Logger.Info("Requested password reset for user that was not found", "user", userQuery.LoginOrEmail)
-		return response.Error(200, "Email sent", err)
+		return response.Error(http.StatusOK, "Email sent", err)
 	}
```

Comment:
Something fishy is happening here: returning Error, with status OK, "Email sent" message and "not found" log. I would like someone to confirm that this is correct.

Reply:
@marefr should we use a non-error response (e.g. not `response.Error`) here? I assume it's for security reasons the response obscures what actually happens.

Reply:
Looks a bit weird but think the reason is that the provided error will be automatically logged. You're correct in we don't want to return the actual problem to user due to security